test/Fortran/fixture-mod/SConstruct

```diff
@@ -0,0 +1,16 @@
+import sys
+
+if sys.platform == "win32":
+    tool_list = ['mingw', 'gfortran', 'gnulink']
+else:
+    tool_list = ['default']
```

Comment:
Can we changed this to `['gfortran','gnulink']`?

Reply:
it's a test, we can do whatever we want :-)